db/migrate/20160721120915_clean_up_pathless_contacts.rb

```diff
@@ -0,0 +1,14 @@
+class CleanUpPathlessContacts < ActiveRecord::Migration
+  def change
+    # There are approx 800 contact ContentItems.
+    ContentItem.find_each(schema_name: "contact", publishing_app: "contacts") do |item|
```

Comment:
Did this work locally? this doesn't seem the correct arguments for [`find_each`](http://api.rubyonrails.org/classes/ActiveRecord/Batches.html#method-i-find_each) looks like it should be `ContentItem.where(schema_name: "contact", publishing_app: "contacts").find_each` ?